### PR TITLE
Add The-First-osu!-Manga-osu!-Combat-Championship

### PR DESCRIPTION
**Pull request by Vortex100K**
The-First-osu!-Manga-osu!-Combat-Championship News Post

### CODE REVIEW

**Review by 0x84f — CHANGES_REQUESTED** [review]
just my thoughts but i think this news post might need more work

File: news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ 
+ Prepare yourselves for a journey like no other as we introduce the highly anticipated osu! Combat Championship (OCC) manga and its thrilling adaptation into an animated series.
+
+ Join us as we delve into the captivating storyline that follows the path of WhiteCat, a skilled player, immersing themselves in a once-in-a-lifetime combat-style tournament where participants battle it out
+ for the ultimate goal: attaining the prestigious rank #1 title and having any wish granted. Get ready to experience exhilarating action sequences, intense rivalries, and delve deeper into our unique spin on the world of osu!
+
+For those eager to dive into the manga, head over to Mangadex and start **reading now!:** [Read OCC Manga on Mangadex](https://mangadex.org/title/f1d50eba-6ace-4490-8439-07692fda3b9c/osu-combat-championship)
+
+But that's not all! We have an exciting announcement for all anime enthusiasts out there. The OCC manga will be transformed into an **animated series**, and we have just started on episode 1's Production!.
+Prepare for action-packed episodes each lasting as long as the animes you love!
```
[the remark this answers is not in the document]
Comment (Vortex100K) [reply news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:21]: This makes it seem as if it's referring to just one which it is not it's referring to all the usual anime released

File: news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+The OCC animated series will faithfully follow the manga, capturing the essence and excitement of a traditional anime experience.
+
+All of the episodes of the OCC Animated Series will be available exclusively on the [VTX YouTube Channel.](https://www.youtube.com/channel/UC7-9V57omhXSVxX9s809n5A) Stay tuned for its release!
+
+## Curious to know who will grace the series with their presence? We are thrilled to feature some of your favorite top players! including
```
[the remark this answers is not in the document]
Comment (Vortex100K) [reply news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:26]: Makes it stand out so the viewer realises that this sentence is more important then the others

File: news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+layout: post
+title: "The First osu! Manga: osu! Combat Championship"
+date: 2023-07-24 13:00:00 +0000
+---
+
+WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted rank #1 title and have his wishes granted. Soon to be an Animated Series!
+
+![](/wiki/shared/news/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship/OCCnewspostrecent.jpg)
+
+**Calling all osu! enthusiasts!**
```
[the remark this answers is not in the document]
Comment (Vortex100K) [reply news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:11]: Then what should i do instead? To start off the article?

[the remark this answers is not in the document]
Comment (0x84f) [reply news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:11]: the start from line 13 is completely fine

File: news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+<p align="center">
+  <img width="95%" height="95%" src="/wiki/shared/news/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship/newbaselist.png">
+</p>
+
+**Now attention to all top 50 players! If you aspire to be part of this series, don't hesitate to reach out to [VTX on any platform!.](https://twitter.com/Vortex100k) Your participation will make this project larger and give excitement to the adaptation.**
+
+## The current team working on this project behind the scenes!
+
+<p align="center">
```
[the remark this answers is not in the document]
Comment (Walavouchey) [reply news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:35]: i'm not sure why this should be a table. the design style of the list doesn't seem bad, but there are still concerns that should be addressable if you have the project files:

- it's still about double the height necessary. there should be at least another column
- the text is centre-aligned within the columns while the profile pills are left-aligned, making it look lopsided
- commas should have a space after them (as it is with "Polishing, 3D" for xootynator)
- "GFX DESIGNER" is randomly in all caps, and "GFX" already exists
- you have "Storyteller" and "Story", what's the difference?
- what's the transparency in the image for?

File: wiki/shared/news/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship/newbaselist.png
Comment (Walavouchey) [inline wiki/shared/news/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship/newbaselist.png:1]: this text has too low contrast against the news page background

image

image

File: news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  <img width="98%" height="98%" src="/wiki/shared/news/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship/OCCTEMremakemin.png">
+</p>
```
Comment (Walavouchey) [inline news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:37]: images are done like this

```suggestion
![](/wiki/shared/news/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship/OCCTEMremakemin.png)
```

File: news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+Prepare for action-packed episodes each lasting as long as the anime you love!
+The OCC animated series will faithfully follow the manga, capturing the essence and excitement of a traditional anime experience.
+
+All of the episodes of the OCC Animated Series will be available exclusively on the [VTX YouTube channel.](https://www.youtube.com/channel/UC7-9V57omhXSVxX9s809n5A) Stay tuned for its release!
+
+## Curious to know who will grace the series with their presence? We are thrilled to feature some of your favorite top players! including
+
+<p align="center">
+  <img width="95%" height="95%" src="/wiki/shared/news/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship/newbaselist.png">
+</p>
```
Comment (Walavouchey) [inline news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:29]:
```suggestion
![](/wiki/shared/news/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship/newbaselist.png)
```

File: news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Walavouchey) [reply news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:25]: the standard is not to misuse headings like this. adding a horizontal line (`---`) right before this line would be more appropriate here

File: news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted rank 1 title and have his wish granted. Soon to be an animated series!
+
+![](/wiki/shared/news/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship/OCCnewspostrecent.jpg)
+
+**Calling all osu! enthusiasts!**
+ 
+ Prepare yourselves for a journey like no other as we introduce the highly anticipated osu! Combat Championship (OCC) manga and its thrilling adaptation into an animated series.
+
+ Join us as we delve into the captivating storyline that follows the path of WhiteCat, a skilled player, immersing themselves in a once-in-a-lifetime combat-style tournament where participants battle it out for the ultimate goal: attaining the prestigious rank 1 title and having any wish granted. Get ready to experience exhilarating action sequences, intense rivalries, and delve deeper into our unique spin on the world of osu!.
```
Comment (Walavouchey) [inline news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:15]:
```suggestion
Join us as we delve into the captivating storyline that follows the path of WhiteCat, a skilled player, immersing themselves in a once-in-a-lifetime combat-style tournament where participants battle it out for the ultimate goal: attaining the prestigious rank 1 title and having any wish granted. Get ready to experience exhilarating action sequences, intense rivalries, and delve deeper into our unique spin on the world of osu!.
```

File: news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+For those eager to dive into the manga, head over to Mangadex and start **[reading now!](https://mangadex.org/title/f1d50eba-6ace-4490-8439-07692fda3b9c/osu-combat-championship)**
+
+But that's not all! We have an exciting announcement for all anime enthusiasts out there. The OCC manga will be transformed into an **animated series**, and we have just started on episode 1's production!
+Prepare for action-packed episodes each lasting as long as the anime you love!
+The OCC animated series will faithfully follow the manga, capturing the essence and excitement of a traditional anime experience.
```
Comment (Walavouchey) [inline news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:21]: paragraphs need to be separated by blank lines

```suggestion
But that's not all! We have an exciting announcement for all anime enthusiasts out there. The OCC manga will be transformed into an **animated series**, and we have just started on episode 1's production!

Prepare for action-packed episodes each lasting as long as the anime you love! The OCC animated series will faithfully follow the manga, capturing the essence and excitement of a traditional anime experience.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
only doing a pass on formatting for today

File: news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (0x84f) [reply news/2023/2023-07-24-The-First-osu!-Manga-osu!-Combat-Championship.md:35]: The height was the main reason that was off-putting for me, the rest I agree with

**Comment by Walavouchey** [comment]
drafting as the direction here seems off as [explained on discord](https://discord.com/channels/188630481301012481/1003689172420276246/1139237961364426832)

**Pull request closed without merging**